rainier-tests/src/test/scala/com/stripe/rainier/compute/RealTest.scala

```diff
@@ -178,6 +178,10 @@ class RealTest extends FunSuite {
       (x * x * x))
   }
 
+  run("x^(10(2/10 + 1/10))") { x =>
```

Comment:
This currently passes, but would fail with a `Decimal(value: Double)` implementation.